docs/usage/configuration.mdx

```diff
@@ -57,48 +57,78 @@ Each module listed under the `modules` key above can accept the following attrib
 - `depends_on`: a list of the other modules which the module is allowed to import from, using their 'paths' to identify them
 - `strict`: enables [strict mode](strict-mode) for the module
 
-## Source Root
+Tach also supports [deprecating individual dependencies](../usage/deprecate).
+
+## Source Roots
 
-The `source_root` key is required for Tach to understand the imports within your project.
-If it is not set explicitly, `source_root` defaults to `'.'`.
-This means Tach will expect that your Python imports are resolved relative to the directory in which `tach.yml` exists (the 'project root').
+The `source_roots` key is required for Tach to understand the imports within your project.
+If it is not set explicitly, `source_roots` defaults to your project root path: `['.']`.
+This means Tach will expect that your Python imports are resolved relative to the directory in which `tach.yml` exists.
 
-Below is a typical case in which `source_root` is necessary.
+Below is a typical case in which modifying `source_roots` is necessary.
 
-### Example
+### Example: Monorepo
```

Comment:
Should this replace the standard example, or should this be in addition to it? I could see monorepo/multiple source root configuration being it's own docs page as it's less likely to represent the path that most take

Reply:
Good idea, restored the previous example